aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -31,6 +43,8 @@ Parameters:
   ConsoleInstanceType:
     Type: String
 <% end -%>
+Conditions:
+  IsProductionCondition: !And [!Equals [!Ref BaseDomainName, "code.org"], !Equals [!Ref DashboardSubDomainName, "studio"]]
```

Comment:
I don't love that this depends on other logic which itself also uses a different method to identify whether or not we're on production. Can we instead just inspect `rack_env?(:production)` here?

Reply:
What does `IsProduction` in this context?

Reply:
Oops. I'm not using that logic anymore and have removed it.